tests/working/005-cse/input

```diff
@@ -1,2 +1,3 @@
 :load main.velo
 :cse
+:quit
```

Comment:
Sorry! I'm used to Idris cleanly quitting when it hits EOF. :D

Reply:
it is my fault! as quitting was added after I wrote the original repl test...